packages/orderbook/scripts/buildPackage.js

```
@@ -67,12 +68,12 @@ esm =
 imports = __wbg_get_imports();
 
 import { Buffer } from 'buffer';
-import wasmB64 from '../${package}.json';
+import wasmB64 from '../orderbook_wbg.json';
 const bytes = Buffer.from(wasmB64.wasm, 'base64');
 
 const wasmModule = new WebAssembly.Module(bytes);
 const wasmInstance = new WebAssembly.Instance(wasmModule, imports);
 wasm = wasmInstance.exports;`;
 esm = esm.replaceAll("imports.wbg", "imports.__wbindgen_placeholder__");
 esm = "/* this file is auto-generated, do not modify */\n" + esm;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)